script/run-gn

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import os
+import sys
+
+import lib.gn as gn
+
+from lib.config import get_output_dir, COMPONENTS, \
+                       SOURCE_ROOT, SRC_DIR, DEPOT_TOOLS_DIR
+
+
+# TODO(alexeykuzmin): There are no reasons to keep args files
+# in the "chromiumcontent" folder.
+CHROMIUMCONTENT_SOURCE_DIR = os.path.join(SOURCE_ROOT, 'chromiumcontent')
+
+
+def parse_args():
+  parser = argparse.ArgumentParser(description='Generate Ninja')
+
+  parser.add_argument('-a', '--args', type=json.loads, default=None,
+                      help='List of GN args in JSON format. E.g. \'{"key": "value"}\'')
+  parser.add_argument('-c', '--component', nargs='+', default=None, choices=COMPONENTS,
+                      help='Component(s) to generate build files for.')
+  parser.add_argument('-t', '--target_arch', default='x64', help='E.g. "x64".')
+
+  return parser.parse_args()
+
+
+def main():
+  args = parse_args()
+
+  # Generate build files for all components if none are specified.
+  components = args.component if args.component else COMPONENTS
```

Comment:
could this logic be folded into the `default` param of the `--component` arg definition?

Reply:
Probably. Thanks!

Reply:
Done